### ISSUE TITLE
[Accessibility]: Plain text is unnecessarily announced as clickable in upload information page

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#44557]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33512

### Problem description

Phrases like 'Directory' and 'indexed at commit' are described as clickable even though they're not clickable.

### Expected behavior

Plain words shouldn't be called out as clickable.

### Additional details

https://user-images.githubusercontent.com/93103176/202493776-8fee4bbd-21eb-4c70-b24d-63526fde5d28.mov

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by felixfbecker** [comment]
I can only reproduce this in Firefox, but I don't see anything wrong with the accessibility tree, so this seems to be a Firefox bug.
